azure-iot-device/samples/advanced-hub-scenarios/provision_symmetric_key_and_send_telemetry.py

```diff
@@ -0,0 +1,69 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
We should also have a sample for symmetric key with group.  And we should figure out how to fold the derive_device_key logic into our api.  I don't like having to force the user to do the hmac and signing for this.  This is something that we should be doing for them.

Reply:
there is a sample already for group but just that the group sample does not include sending telemetry. are you referring to a group sample that will also send telemetry ?

on a separate note , in the group sample the derive device key is part of the sample. as per my understanding the derivation of device key is something that customers should do but i can verify once with jelani.

---
In reply to: [354668549](https://github.com/Azure/azure-iot-sdk-python/pull/407#discussion_r354668549) [](ancestors = 354668549)